packages/deprecated/src/index.ts

```diff
@@ -61,15 +83,9 @@ export default function deprecated( feature, options = {} ) {
 	/**
 	 * Fires whenever a deprecated feature is encountered
 	 *
-	 * @param {string}  feature             Name of the deprecated feature.
-	 * @param {?Object} options             Personalisation options
-	 * @param {string}  options.since       Version in which the feature was deprecated.
-	 * @param {?string} options.version     Version in which the feature will be removed.
-	 * @param {?string} options.alternative Feature to use instead
-	 * @param {?string} options.plugin      Plugin name if it's a plugin feature
-	 * @param {?string} options.link        Link to documentation
-	 * @param {?string} options.hint        Additional message to help transition away from the deprecated feature.
-	 * @param {?string} message             Message sent to console.warn
+	 * @param {string}             feature Name of the deprecated feature.
+	 * @param {?DeprecatedOptions} options Personalisation options
+	 * @param {?string}            message The deprecation message.
```

Comment:
None of the params here is optional here. Let us fix that

```suggestion
	 * @param {string}            feature Name of the deprecated feature.
	 * @param {DeprecatedOptions} options Personalisation options
	 * @param {string}            message Message sent to console.warn
```